configs/nas/greedynas/greedynas_supernet_mobilenet_8xb128_in1k.py

```diff
@@ -0,0 +1,162 @@
+_base_ = [
+    '../../_base_/datasets/mmcls/imagenet_bs128_colorjittor_greedynas.py',
+    '../../_base_/schedules/mmcls/imagenet_bs1024_greedynas.py',
+    '../../_base_/mmcls_runtime.py'
+]
+
+norm_cfg = dict(type='BN')
+model = dict(
+    type='mmcls.ImageClassifier',
+    backbone=dict(
+        type='SearchableMobileNet',
+        arch_setting_type='greedynas',
+        widen_factor=1.0,
+        norm_cfg=norm_cfg,
+        act_cfg=dict(type='ReLU')),
```

Comment:
I notice that all activation layers in searchable block used `HSwish`. However, `act_cfg` here used `ReLU`, which will affect the first and last `ConvModule` in mobilenet.
Is that expected?

Reply:
yes, it is confirmed with huangtao